instrumentation/logback/logback-appender-1.0/library/build.gradle.kts

```diff
@@ -35,3 +36,18 @@ dependencies {
   testImplementation("io.opentelemetry:opentelemetry-sdk-logs")
   testImplementation("io.opentelemetry:opentelemetry-sdk-testing")
 }
+
+graalvmNative {
+
+  binaries.all {
+    resources.autodetect()
+  }
+
+  toolchainDetection.set(false)
+}
+
+configurations.configureEach {
+  exclude("org.apache.groovy", "groovy")
```

Comment:
can you add a brief comment explaining why we need to exclude these even though there aren't being used?